contracts/protocols/mainnet/compound_iii/main.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+import "./interfaces.sol";
+import "./helpers.sol";
+
+/**
+ *@title Compund III Resolver
+ *@dev get user position, user configuration, market configuration.
+ */
+contract CompoundIIIResolver is CompoundIIIHelpers {
```

Comment:
@Richa-iitr Is there a way to get all the active markets?

Reply:
Couldn't find anything in the contracts which returns the addresses of the markets. There's just [clone](https://github.com/compound-finance/comet/blob/main/contracts/CometFactory.sol#L8) method which returns the address on clone, we might track this.

Reply:
@Richa-iitr Can you share github link how is this clone deployed?

Reply:
@thrilok209 clone deployment https://github.com/compound-finance/comet/blob/main/contracts/CometFactory.sol

Reply:
@Richa-iitr @pradyuman-verma This just deploys the implementation, right? But not the main proxy contract. https://github.com/compound-finance/comet/blob/main/contracts/Configurator.sol#L334

Reply:
Main proxy contract is cloned.
And the configurator deploys the implementations for the proxies